### ISSUE TITLE
[FEATURE] Adding name, group, description to KeyMap

### ISSUE THREAD

**Issue by huv1k** [greena13/react-hotkeys#154]
I am missing description for `KeyMap`, this could work great with `getApplicationKeyMap` building dynamic keymap. Lot of websites show keymap with combination `shift+?`. For example github example: ![](https://i.imgur.com/7QfYIFl.png)

## Possible solution
```js
const keyMap = {
  SUBMIT_COMMET: {
    name: 'Submit comment',
    group: 'Issue',
    description: 'Submit comment',
    sequence: 'command+enter'
  }
}
```

**Comment by greena13** [comment]
Hi @Huvik, this is an excellent idea and matches up with some future plans I had for the library. 

I will try and add it into the next release.

**Comment by Huvik** [comment]
This is only one thing I am missing right now:)

**Comment by greena13** [comment]
This has been added and will be available in the next release.

**Comment by greena13** [comment]
This feature is now available in [v2.0.0-pre8](https://github.com/greena13/react-hotkeys/releases/tag/v2.0.0-pre8).
